VimFlavor

```diff
@@ -0,0 +1 @@
+flavor 'fatih/vim-go', '~> 1.5'
```

Comment:
Good call. Btw I've changed the source code a lot and renamed the internal functions too. They will not work and probably will be broken. But I'm going to work on tests soon.